gcp/modules/couchdb/main.tf

```diff
@@ -121,3 +121,17 @@ resource "null_resource" "couchdb_destroy_pvcs" {
     EOF
   }
 }
+
+
+resource "null_resource" "couchdb_resources" {
+  depends_on = ["module.couchdb"]
+
+  provisioner "local-exec" {
+    command = "kubectl apply -f ${path.module}/resources/"
+  }
```

Comment:
This will not reapply if the policy file is changed, will it?

Reply:
You're right, it won't currently. We have a number of places in our code base that do similar things. I  believe the solution is to add `triggers` based on the checksum of the files in the directory. I'll implement it this way unless there are any concerns with that approach.

Reply:
I think triggers are the right solution. I think @natarajaya has some experience with this.

Reply:
@seg10 Check this one:
https://github.com/gpii-ops/gpii-infra/blob/master/gcp/modules/couchdb/main.tf#L90